appeals/web/src/server/appeals/appeal-details/start-case/__tests__/start-case.test.js

```diff
@@ -13,27 +13,51 @@ const appealDataWithoutStartDate = {
 };
 
 describe('start-case', () => {
-	beforeEach(() => {
-		nock('http://test/').get('/appeals/1').reply(200, appealDataWithoutStartDate);
-	});
 	afterEach(teardown);
 
 	describe('GET /start-case/add', () => {
 		it('"Start case" page should contain "Confirming will activate the timetable on"', async () => {
+			nock('http://test/').get('/appeals/1').reply(200, appealDataWithoutStartDate);
 			const response = await request.get(`${baseUrl}/1/start-case/add`);
 			expect(response.text).toContain('Confirming will activate the timetable on');
 		});
 
 		it('should redirect to the confirmation page', async () => {
+			nock('http://test/').get('/appeals/1').reply(200, appealDataWithoutStartDate);
 			nock('http://test/').post(`/appeals/1/appeal-timetables`).reply(200, {});
 			const response = await request.post(`${baseUrl}/1/start-case/add`).send({});
 			expect(response.statusCode).toBe(302);
 		});
 	});
 
-	describe('GET /start-case/confirmation', () => {
+	describe('GET /start-case/add/confirmation', () => {
 		it('should render "Case started" confirmation page', async () => {
-			const response = await request.get(`${baseUrl}/1/start-case/confirmation`);
+			nock('http://test/').get('/appeals/1').reply(200, appealDataWithoutStartDate);
+			const response = await request.get(`${baseUrl}/1/start-case/add/confirmation`);
+			const element = parseHtml(response.text);
+			expect(element.innerHTML).toMatchSnapshot();
+		});
+	});
+
+	describe('GET /start-case/change', () => {
+		it('"Start case" page should contain "Confirming will change the start day to"', async () => {
+			nock('http://test/').get('/appeals/1').reply(200, appealData);
+			const response = await request.get(`${baseUrl}/1/start-case/change`);
+			expect(response.text).toContain('Confirming will change the start day to');
+		});
+
+		it('should redirect to the confirmation page', async () => {
+			nock('http://test/').get('/appeals/1').reply(200, appealData);
+			nock('http://test/').post(`/appeals/1/appeal-timetables`).reply(200, {});
+			const response = await request.post(`${baseUrl}/1/start-case/change`).send({});
+			expect(response.statusCode).toBe(302);
+		});
+	});
+
+	describe('GET /start-case/change/confirmation', () => {
+		it('should render "Start date changed" confirmation page', async () => {
+			nock('http://test/').get('/appeals/1').reply(200, appealData);
+			const response = await request.get(`${baseUrl}/1/start-case/change/confirmation`);
```

Comment:
As above, you could add checks for specific content in the snapshot here

Reply:
done